docs/trex_release-notes.md

```diff
@@ -11,9 +11,44 @@ layout: docs
 ----
 See also: [Known Issues]({{site.baseurl}}/docs/trex_known_issues.html)
 
+----
+
+### Tableau Dashboard Extensions API version 1.5
+
+*June 2021*
+
+* Tableau Dashboard Extensions API library: `tableau.extensions.1.5.0.js` <br>(download or clone the Extensions API repository on [GitHub](https://github.com/tableau/extensions-api){:target="_blank"}.) <br/>
+
+About this release:
+
+* The `Filter.getFieldAsync` method now works as expected and properly returns the field. This fix requires Tableau 2019.2 and later.
+
+* Show/Hide (`setZoneVisibilityAsync`) can now be applied to any dashboard zone.
+
+* The `selectMarksByValueAsync` method now supports combined selection criteria types (bug fixed).
+
+* The following are all improvements to the `getSummaryData` method:
+
+  * `getSummaryData` now has a smaller and faster payload.
+
+  * `maxRows` can be applied to `getSummaryData` to restrict the number of rows fetched.
+
+  * `columnsToIncludeById` can be applied to `getSummaryData` to restrict the columns fetched.
+
+  * `includeDataValuesOnly` or `includeFormattedValuesOnly` can be applied to `getSummaryData` to restrict the amount of information returned to what you really need.
+
+  * The `getSummaryColumnsInfoAsync` method is a new in this release. It returns the column information for each column that will be returned in `getSummaryData`.
```

Comment:
"method is a new in this release". There is a word or two missing, or just in the wrong order.

Reply:
yes. There are words missing there. Thanks for catching that.   I think it was supposed to say:


The `getSummaryColumnsInfoAsync` method is new in this release. It returns the column information for each column that will be returned in `getSummaryDataAsync`.